docs/references/glossary.md

```diff
@@ -0,0 +1,328 @@
+
+## Backstage
+
+A platform for creating and deploying [developer portals](#developer-portal), originally created at Spotify.
+
+Backstage is an incubation-stage open source project of the [Cloud Native Computing Foundation](#cloud-native-computing-foundation).
+
+Can also refer to: [Backstage framework](#backstage-framework).
+
+## Backstage framework
+
+The actual framework that Backstage [plugins](#plugin) sit on. This spans both the frontend and the backend, and includes core functionality such as declarative integration, config reading, database management, and many more.
+
+## Bundle
+
+A collection of [deployment artifacts](#deployment-artifacts).
+
+Can also be: The output of the bundling process, which brings a collection of [packages](#package) into a single collection of [deployment artifacts](#deployment-artifacts).
+
+## Catalog
+
+An organization's portfolio of software products managed in Backstage.
+
+Can also be: The core Backstage plugin that handle ingestion and display of your organizations software products.
+
+## Cloud Native Computing
+
+A set of technologies that "empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach." ([CNCF Cloud Native Definition v1.0](https://github.com/cncf/toc/blob/main/DEFINITION.md)).
+
+## Cloud Native Computing Foundation
+
+A foundation dedicated to the promotion and advancement of [Cloud Native Computing](#Cloud-Native-Computing). The mission of the Cloud Native Computing Foundation (CNCF) is "to make cloud native computing ubiquitous" ([CNCF Charter](https://github.com/cncf/foundation/blob/main/charter.md)).
+
+CNCF is part of the [Linux Foundation](https://www.linuxfoundation.org/).
+
+## CNCF
+
+Cloud Native Computing Foundation.
+
+## Code Grant
+
+[OAuth](#oauth) flow where the client receives an [authorization code](#code) that is passed to the backend to be exchanged for an [access token](#access-token) and possibly a [refresh token](#refresh-token).
+
+## Collators (search plugin)
+
+Collators transform streams of [documents](#documents) into searchable texts. They're usually responsible for the data transformation and definition and collection process for specific [documents](#documents). Part of [Backstage Search](#search).
+
+## Component (catalog plugin)
+
+A software product that is managed in the Backstage [Software Catalog](#software-catalog). A component can be a service, website, library, data pipeline, or any other piece of software managed as a single project.
+
+https://backstage.io/docs/features/software-catalog/system-model
+
+## Condition (permission plugin)
+
+Conditions are used to return a conditional decision from a policy. They contain information about a given entity and restrictions on what types of users can view that entity.
```

Comment:
```suggestion
A declaration that returns a conditional decision from a policy. Suggestions contain information about a given entity and restrictions on what types of users can view that entity.
```

In general, I recommend trying to make the first sentence of a glossary entry a definition of what the entry *is* rather than what it *does*. You can then expand on the definition if necessary. If the definition becomes too complex, it might be that it's actually documentation that belongs elsewhere in the doc.

Reply:
Definitely agree, this will definitely be something we'll want to iterate on to get the right wording. I updated a couple of the entries that were definitely too wordy. I think pointing back to a plugin or framework doc would help a lot here.